forest/poplar/legacy/tests/test_poplar_functions.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+import numpy as np
+import pandas as pd
+from tempfile import TemporaryDirectory
+
+from forest.poplar.legacy.common_funcs import (
+    read_data, datetime2stamp, stamp2datetime, write_all_summaries,
+    get_files_timestamps, filename2stamp
+)
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_DATA_DIR = os.path.join(CURRENT_DIR, "test_data")
+
+
+def test_datetime2stamp():
+    time_list = [2020, 11, 1, 12, 9, 50]
+    assert datetime2stamp(time_list=time_list,
+                          tz_str="America/New_York") == 1604250590
+    assert datetime2stamp(time_list=time_list,
+                          tz_str="America/Chicago") == 1604254190
+
+
+def test_datetime2stamp_bad_values():
+    second_error_caught = False
+    minute_error_caught = False
+    hour_error_caught = False
+    day_error_caught = False
+    month_error_caught = False
+    try:
```

Comment:
[pytest.raises()](https://docs.pytest.org/en/stable/reference/reference.html?highlight=raises#pytest.raises) is what you need here. Also, there should be a separate test for each type of error.